app/routes/events/view/tickets/access-codes/create.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+import moment from 'moment';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    return this.l10n.t('Create');
+  },
+  model() {
+    return {
+      code          : '',
+      isActive      : true,
+      ticketsNumber : 10,
+      minQuantity   : 1,
+      maxQuantity   : 10,
+      validFromDate : moment(),
+      validFromTime : '12:00',
+      validTillDate : moment().add(20, 'days'),
+      validTillTime : '16:00',
+      tickets       : ['Early bird', 'Late entry']
```

Comment:
Why do you need to display ID ? I don't understand. IDs are always for internal reference.